egl/egl_util.c

```diff
@@ -30,6 +30,26 @@
 #include "libdrm/drm_fourcc.h"
 #endif
 
+static PFNEGLCREATEIMAGEKHRPROC createImageProc = NULL;
+static PFNEGLDESTROYIMAGEKHRPROC destroyImageProc = NULL;
+
+EGLImageKHR createImage(EGLDisplay dpy, EGLContext ctx, EGLenum target,
+                        EGLClientBuffer buffer, const EGLint *attrib_list)
+{
+    if (!createImageProc) {
```

Comment:
This need run time check and did not handle error for eglGetProcAddress returns.
Could we init createImageProc,destroyImageProc & imageTargetTexture2D, in one place and check the return value?